api/mindexd/store/query.go

```diff
@@ -0,0 +1,53 @@
+package store
```

Comment:
Would it be useful to add some things to the `hub` CLI in order to tease out some of the needed APIs? Might also be nice for users to poke around at all the rich data.

Reply:
The main way of interaction for this data it's going to be a web dashboard to explore it. I'd imagine that web interface would be much easier and powerful than any CLI commands that we might come up with.
So, I think makes sense to make that web UI shape the "Query API" of all this... if we conclude that some people would like to use CLI commands instead of that web UI to explore this data, we can add them I guess.